addons/mass_mailing_sms/wizard/mailing_sms_test.py

```diff
@@ -46,7 +46,8 @@ def action_send_sms(self):
 
         for sent_sms, db_sms in zip(sent_sms_list, new_sms_messages_sudo):
             recipient = db_sms.number or sent_sms.get('res_id')
-            if sent_sms.get('state') == 'success':
+            # 'success' and 'sent' both resolve to 'send' SMS state via IAP_TO_SMS_STATE_SUCCESS
```

Comment:
Suggestion to limit further confusion
```suggestion
            # 'success' and 'sent' both resolve to 'pending' SMS state (=sent) via IAP_TO_SMS_STATE_SUCCESS
```